README.md

```diff
@@ -73,22 +74,20 @@ Latest releases:
 * [3.1.1](https://esl.github.io/MongooseDocs/3.1.1/)
 * [3.0.1](https://esl.github.io/MongooseDocs/3.0.1/)
 
-
 **MongooseIM documentation highlights:**
 
 When developing new features/modules, please make sure you add basic documentation to the 'doc/' directory, and add a link to your document in 'doc/README.md.'
 
 * [Tutorials](https://esl.github.io/MongooseDocs/latest/user-guide/How-to-build/). Learn how to:
-    * [Build MongooseIM from source code](doc/user-guide/How-to-build.md)
-    * [Set up MongoosePush](doc/user-guide/push-notifications/Push-notifications.md)
-    * [Set up MongooseICE](doc/user-guide/ICE_tutorial.md)
-    * [Build an iOS messaging app](doc/user-guide/iOS_tutorial.md)
-* [User Guide](https://esl.github.io/MongooseDocs/latest/user-guide/ABCs-of-MongooseIM/). Learn all about how to use MongooseIM in your project. Explore its features, supported XEPs, RFCs and database backends, as well as its architecture and deployment strategies.
-* [Configuration](https://esl.github.io/MongooseDocs/latest/Basic-configuration/). Explore some of the available options including database backend configuration, access control lists, listener and extension modules.
+    * [Build MongooseIM from source code](https://esl.github.io/MongooseDocs/latest/user-guide/How-to-build.md)
```

Comment:
Not sure that the use link https://esl.github.io/MongooseDocs/latest is correct, cuz it's instead of `latest` will be used added `x.x` tag version and eg for `5.0` and for `5.2` links can be changed or removed etc. Please use relative path as it was before.